server/cdk/lib/bootstrap-template/bootstrap-template-stack.ts

```diff
@@ -0,0 +1,137 @@
+import { Stack, StackProps, } from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+import { ApiKeySSMParameterNames } from '../interfaces/api-key-ssm-parameter-names';
+import { EventBus } from "aws-cdk-lib/aws-events";
+import { TenantApiKey } from './tenant-api-key';
+import { Table, AttributeType } from 'aws-cdk-lib/aws-dynamodb';
+import { PolicyDocument } from 'aws-cdk-lib/aws-iam';
+import { CoreApplicationPlane, DetailType, EventManager } from '@cdklabs/sbt-aws';
+import * as fs from 'fs';
+
+interface BootstrapTemplateStackProps extends StackProps {
+  ApiKeySSMParameterNames: ApiKeySSMParameterNames;
+  apiKeyPlatinumTierParameter: string;
+  apiKeyPremiumTierParameter: string;
+  apiKeyStandardTierParameter: string;
+  apiKeyBasicTierParameter: string;
+  eventBusArn: string;
+  systemAdminEmail: string;
+}
+
+export class BootstrapTemplateStack extends Stack {
+  public readonly tenantMappingTable: Table;
+
+  constructor(scope: Construct, id: string, props: BootstrapTemplateStackProps) {
+    super(scope, id, props);
+
+    const systemAdminEmail = props.systemAdminEmail;
+    const eventBusArn = props.eventBusArn;
+
+    const eventBus = EventBus.fromEventBusArn(this, 'EventBus', eventBusArn);
+    const eventManager = new EventManager(this, 'EventManager', {
+      eventBus: eventBus,
+    });
+    this.tenantMappingTable = new Table(this, 'TenantMappingTable', {
+      partitionKey: {name: 'tenantId', type: AttributeType.STRING},
+    });
+
+    const provisioningJobRunnerProps = {
+      name: 'provisioning',
+      permissions: PolicyDocument.fromJson(
+        JSON.parse(`
+{
+  "Version":"2012-10-17",
```

Comment:
add a comment to scope this as a best practice

Reply:
Comment added.